src/instructlab/sdg/generate_data.py

```diff
@@ -204,6 +204,7 @@ def generate_data(
     tls_client_passwd: Optional[str] = None,
     # TODO need to update the CLI to specify which pipeline to use (simple or full at the moment)
     pipeline: Optional[str] = "simple",
+    tls_ca_bundle: Optional[str] = None,
```

Comment:
I had to look up the httpx.Client API docs to find out for sure what valuer was supposed to be used here (a path to a CA bundle). Could you add a comment above this or in a docstring that clarifies this?  You could link to https://www.python-httpx.org/advanced/ssl/ which is where I found the answer.

Reply:
updated with comment!